strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/npm/NpmArtifactController.java

```diff
@@ -394,6 +395,17 @@ public ResponseEntity publish(@RepositoryMapping Repository repository,
         return ResponseEntity.ok("");
     }
 
+    @PreAuthorize("hasAuthority('UI_LOGIN')")
+    @PutMapping(path = "{storageId}/{repositoryId}/-/user/org.couchdb.user:{username}",
```

Comment:
Just out of curiousity... Are we sure about the `org.couchdb.user:{username}` string...? I mean -- is this the same behaviour for Verdaccio and, say npmjs.org, (or some other npm server)?

Reply:
this is coming from the npm application. I saw it as well in verdaccio.

Reply:
Okay, cool! :)

Thanks for confirming!